### PR TITLE
Add new post and new comment form

### PR DESCRIPTION
**Pull request by awaisanwar544**
Add forms:

1. Add new post for the user
2. Add new comment to the post
3. Like a post

### CODE REVIEW

File: app/controllers/service_controller.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    @current_user = ApplicationController.current_user
+  end
+
+  def like
+    user = User.find(params[:user_id])
+    post = Post.find(params[:post_id])
+    Like.create(user:, post:)
+    Like.update_counter_for_post(post)
+    redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
+  end
```
Comment (elisha2kyakpo1) [inline app/controllers/service_controller.rb:13]: - [ ] Kindly fix an error in the likes action, I couldn't create a like or a comment. See the error in the image below.

likes

File: app/controllers/service_controller.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  def comment
+    post = Post.find(params[:post_id])
+    text = params[:text]
+    if text == ''
+      redirect_to user_post_path(user_id: params[:user_id], id: params[:post_id])
+      return
+    end
+    Comment.create(user: @current_user, post:, text:)
```
Comment (elisha2kyakpo1) [inline app/controllers/service_controller.rb:22]: - [ ] The same can be done on this line of code. It was giving me an error like the one in the likes action.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @awaisanwar544, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The PR title and description look excellent and professional 👍
- The readme file is well documented 💯
- No linter errors 👍

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- It's a good practice to commit often for every change we make to the code.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by awaisanwar544** [comment]
@elisha2kyakpo1 I am unable to recreate the error because it's working fine for me. I can like and comment please watch the video


https://user-images.githubusercontent.com/87834222/173191399-42dee2ce-3808-47b9-a720-0e8d8c3d2684.mov

File: app/controllers/service_controller.rb (hunk 1 above)
Comment (elisha2kyakpo1) [reply app/controllers/service_controller.rb:13]: @awaisanwar544, I'm sure the comma on line 10. Like.create(user:, post:)

**Review by keneogu — APPROVED** [review]
> ## Hi @awaisanwar544,
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> 
> ## Congratulations!     
> Cheers and Happy coding!clapclapclap
> 
> image [ ](https://camo.githubusercontent.com/2366be437d35bce33dd123620530e9131a8ac147fe960e76a36614dc703e8e22/68747470733a2f2f692e70696e696d672e636f6d2f6f726967696e616c732f37372f30652f62302f37373065623035373434393538326334343637326232643264316363333962362e676966) image [ ](https://camo.githubusercontent.com/2366be437d35bce33dd123620530e9131a8ac147fe960e76a36614dc703e8e22/68747470733a2f2f692e70696e696d672e636f6d2f6f726967696e616c732f37372f30652f62302f37373065623035373434393538326334343637326232643264316363333962362e676966)
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**